docker-compose/send-sample-metrics.py

```diff
@@ -0,0 +1,114 @@
+        # send_json_to_kafka()              # Send to kafka using JSON format
+    producer.flush()
+
+    print("Checking for anomalies from Kafka...\n")
+    weak_anomaly_values, strong_anomaly_values = consume_from_kafka('anomalies')
+    print("\n-- Found", len(weak_anomaly_values), "WEAK Anomalies")
+    print("-- Weak Anomalous values:", weak_anomaly_values)
+    print("\n-- Found", len(strong_anomaly_values), "STRONG Anomalies")
+    print("-- Strong Anomalous values:", strong_anomaly_values)
+    
+
+# Default Msgpack Serde - Use if ad-mapper.conf is set to "com.expedia.adaptivealerting.kafka.serde.MetricDataMessagePackSerde"
+def send_msgpack_to_kafka(producer, topic):
+    metric_data = {
+        "OrgId": 1, 
+        "Name": "samplemetric", 
+        "Interval": 15,
+        "Value": round(random.uniform(10,200), 1),
+        "Unit": "unknown",
+        "Time": int(time.time()),
+        "Mtype": "gauge",
+        "Tags": ["what=samplemetric", "m_application=sample-web", "region=primary", "env=prod"]
+    }
+    packed_metric_data = msgpack.packb(metric_data, use_bin_type=True)
+    producer.send(topic, key=b'prod.primary.sample-web.sample-metric', value=packed_metric_data)
+
+
+# JSON Serde - Use if ad-mapper.conf is set to "com.expedia.adaptivealerting.kafka.serde.MetricDataJsonSerde"
+def send_json_to_kafka(producer, topic):
+    metric_data = {
+                    "metricDefinition": {
+                        "key": "samplemetric",
+                        "tags": {
+                            "kv": {
+                                "what": "samplemetric",
+                                "m_application": "sample-web",
+                                "region": "primary",
+                                "env": "prod"
+                            },
+                            "v": []
+                        },
+                        "meta": {
+                            "kv": {},
+                            "v": []
+                        }
+                    },
+                    "value": round(random.uniform(10,200), 1),
+                    "timestamp": int(time.time())
+                }
+    metric_data = json.dumps(metric_data).encode()
+    producer.send(topic, key=b'prod.primary.sample-web.sample-metric', value=metric_data)
+
+
+def consume_from_kafka(topic):
+    output_count = 0
+    anomaly_count = 0
+    strong_values = []
+    weak_values = []
+    consumer = KafkaConsumer(topic,
+                            bootstrap_servers=kafka_boostrap_servers, 
+                            auto_offset_reset='earliest', 
+                            enable_auto_commit=False, 
+                            consumer_timeout_ms=5000)
+    
+    for message in consumer:
+        output_count+=1
+        anomaly = json.loads(message.value)
+        anomalyLevel = anomaly.get('anomalyResult',{}).get('anomalyLevel')
+        if anomalyLevel == 'WEAK':
```

Comment:
anomaly_value = {}
if anomalyLevel == 'WEAK' or anomalyLevel == 'STRONG'
    anomaly_count+=1
     anomaly_value[anomalyLevel] = anomaly.get('metricData').get('value')
return anomaly_value


and use weak_anomaly_values = anomaly_value['WEAK']